app/model/editions/templates/DailyEdition.scala

```diff
@@ -7,7 +7,14 @@ import model.editions._
 import model.editions.templates.TemplateHelpers._
 
 //noinspection TypeAnnotation
-object DailyEdition {
+object DailyEdition extends EditionDefinitionWithTemplate {
+  override val title = "The Daily"
+  override val subTitle = "Published every morning by 6am (GMT)"
+  override val edition = "daily-edition"
```

Comment:
We already know this is an edition def. from the object name, but the name `edition` doesn't tell us much about the use of this field (although the string type gives you a hint) – for developers, is this a decorative `name`, or a uniquely referenced `id`? etc? Guessing the latter, in which case would `editionId` or just `id` be preferable?

Reply:
It's the 'priority' or part of the 'key'... I don't quite know what to call it.